packages/api/src/externals/function.rs

```diff
@@ -199,9 +199,7 @@ impl Function {
                 let results = environment
                     .py_function
                     .call(py, PyTuple::new(py, arguments), None)
-                    .map_err(|error| {
-                        wasmer::RuntimeError::new(io::Error::from(error).to_string())
-                    })?;
+                    .map_err(|error| wasmer::RuntimeError::custom(Box::new(error)))?;
```

Comment:
Probably not need to map it any longer?

Reply:
This is necessary to pass the `PyErr` through the `RuntimeError` in wasmer. The closure passed to `Function::new_with_env` must have an error type of `RuntimeError`.